api/models/data_populator.py

```diff
@@ -0,0 +1,28 @@
+import abc
+
+from django.db.backends.sqlite3.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+class DataPopulator(abc.ABC):
+    """
+    Abstract class-based variation used to make data migrations.
```

Comment:
```suggestion
    Abstract class-based variation used to apply data migrations.
```
(to discern `makemigrations` from `migrate`. This populator thing is for the `migrate` stage.